toonz/sources/common/tiio/compatibility/tfile_io.c

```diff
@@ -58,7 +58,7 @@ char *convertWCHAR2CHAR(const wchar_t *wc) {
 }
 #endif
 /*-----------------------------------*/
-#if defined(MACOSX) || defined(LINUX)
+#if defined(MACOSX) || defined(LINUX) || defined(FREEBSD)
```

Comment:
Should a `UNIX` macro be defined? I'm not sure how useful it'd be.

Reply:
```UNIX``` definition not used in OpenToonz.